### PR TITLE
semantic aside first draft

### PR DESCRIPTION
**Pull request by short-matthew-f**

### CODE REVIEW

File: content/html/glossaries/semantic-html/aside.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+Title: "<aside>"
```
Comment (sonnynomnom) [inline content/html/glossaries/semantic-html/aside.md:2]: Solid metadata!

File: content/html/glossaries/semantic-html/aside.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+</aside>
+``` 
+
+`<aside>` is a tag that wraps around a block of HTML giving it semantic meaning. It has no special attributes, and by default will act similarly to a `<div>`.
+
+## Example
+
+Suppose the current page wants to feature a definition of an important term without having it be a part of the text the term appears in:
+
+```html
```
Comment (sonnynomnom) [inline content/html/glossaries/semantic-html/aside.md:32]: Should we add a full file? With:

```html
<html>
  <head>
  </head>
  <body>
     Move the code here?
  </body>
</html>
```

Comment (sonnynomnom) [reply content/html/glossaries/semantic-html/aside.md:32]: Let's also take out the `<em>`s to not distract.

**Review by sonnynomnom — COMMENTED** [review]
Solid!

File: content/html/glossaries/semantic-html/aside.md (hunk 2 above)
Comment (short-matthew-f) [reply content/html/glossaries/semantic-html/aside.md:32]: Done

**Comment by short-matthew-f** [comment]
Ready for review

**Pull request merged**